src/app/fdctl/run/tiles/fd_plugin.c

```diff
@@ -85,7 +85,7 @@ after_frag( fd_plugin_ctx_t *   ctx,
   switch( in_idx ) {
     /* replay_plugin */
     case 0UL: {
-      FD_TEST( sig==FD_PLUGIN_MSG_SLOT_ROOTED || sig==FD_PLUGIN_MSG_SLOT_OPTIMISTICALLY_CONFIRMED || sig==FD_PLUGIN_MSG_SLOT_COMPLETED || sig==FD_PLUGIN_MSG_SLOT_RESET || sig==FD_PLUGIN_MSG_START_PROGRESS );
+      FD_TEST( sig==FD_PLUGIN_MSG_SLOT_ROOTED || sig==FD_PLUGIN_MSG_SLOT_OPTIMISTICALLY_CONFIRMED || sig==FD_PLUGIN_MSG_SLOT_COMPLETED || sig==FD_PLUGIN_MSG_SLOT_RESET || sig==FD_PLUGIN_MSG_START_PROGRESS || sig==FD_PLUGIN_MSG_GENESIS_HASH_KNOWN);
```

Comment:
Annoying style nits for first PR or two :)

```suggestion
      FD_TEST( sig==FD_PLUGIN_MSG_SLOT_ROOTED || sig==FD_PLUGIN_MSG_SLOT_OPTIMISTICALLY_CONFIRMED || sig==FD_PLUGIN_MSG_SLOT_COMPLETED || sig==FD_PLUGIN_MSG_SLOT_RESET || sig==FD_PLUGIN_MSG_START_PROGRESS || sig==FD_PLUGIN_MSG_GENESIS_HASH_KNOWN );
```